apps/docs/package.json

```diff
@@ -7,6 +7,7 @@
   "scripts": {
     "dev": "vitepress",
     "build": "vitepress build",
+    "build:devhub": "VITEPRESS_BASE=/frontends/ vitepress build",
```

Comment:
@bojanrajh I added the usage of `withBase()` function of vitepress to enable you and us to control the prefix in URL. since now on we can continue to provide an absolute path for PageRefs globally.

that fixes the problem in both environments - dev portal and our standalone docs.

cc: @patzick